src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -1,25 +1,31 @@
 package seedu.address.logic.commands;
 
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.function.Predicate;
+
+import seedu.address.model.person.ReadOnlyPerson;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
- * Keyword matching is case sensitive.
+ * Finds and lists all persons in address book who meet the specified criteria.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
     public static final String COMMAND_ALIAS = "f";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons who meet the specified criteria"
+            + "and displays them as a list with index numbers.\n"
             + "Alias: " + COMMAND_ALIAS + "\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Options: \n"
+            + "\tdefault - Find contacts whose names contain any of the specified keywords (case-sensitive)\n"
+            + "\t" + PREFIX_TAG + " - Find contacts whose tags contain any of the specified keywords (case-sensitive)\n"
+            + "Parameters: [OPTION] KEYWORD [MORE_KEYWORDS]...\n"
```

Comment:
Should it be `KEYWORD [OPTION] [MORE_KEYWORDS]` instead? 🤔

Reply:
I still think it should be [OPTION] KEYWORD [MORE_KEYWORDS].
For instance `find t/(Option) friends(Keyword) colleagues(Another keyword)`

Reply:
Oh right, they are parameters! My mistake, I thought KEYWORD was referring to COMMAND_WORD. Drowsy me. All is well, time to merge your PR too! 👍